Userland/Libraries/LibGfx/Rect.h

```diff
@@ -657,7 +657,7 @@ class Rect {
         };
 
         check_distance({ top_left(), top_right() });
```

Comment:
Shouldn't this `top_right()` receive a `.moved_left(1)` as well?

Reply:
Yep,  you were right. Added a testcase and it was crashing with the point in the top right corner. Fixed now and pushed.